baremetalpxeserver/baremetalpxeserver/pxeserveragent.py

```
@@ -491,17 +548,19 @@ def _create_pxelinux_cfg(self, cmd):
             append = 'interface=auto auto=true priority=critical url=ftp://{PXESERVER_DHCP_NIC_IP}/ks/{KS_CFG_NAME}'
         elif cmd.preconfigurationType == 'autoyast':
             append = 'install=ftp://{PXESERVER_DHCP_NIC_IP}/{IMAGEUUID}/ autoyast=ftp://{PXESERVER_DHCP_NIC_IP}/ks/{KS_CFG_NAME} vnc=1 vncpassword=password'
-        append = append.format(PXESERVER_DHCP_NIC_IP=pxeserver_dhcp_nic_ip,
-                IMAGEUUID=cmd.imageUuid,
-                KS_CFG_NAME=ks_cfg_name)
+        elif cmd.preconfigurationType == 'autoinstall':
+            append = 'ip=dhcp url=ftp://{PXESERVER_DHCP_NIC_IP}/{IMAGEUUID} autoinstall ds=nocloud-net\;s=https//{PXESERVER_DHCP_NIC_IP}:7773/ks/{KS_CFG_NAME}/ ---'
```

Ruff (0.15.15)

[warning] Invalid escape sequence: `\;`

Use a raw string literal

(W605)